cli/src/cmd/forge/create.rs

```diff
@@ -18,6 +18,7 @@ use std::fs;
 use crate::{compile, opts::forge::ContractInfo};
 use clap::{Parser, ValueHint};
 use std::{path::PathBuf, sync::Arc};
+use serde_json::{json, to_string};
```

Comment:
You won't need to import `to_string` then

```suggestion
use serde_json::json;
```